sycl/test/sub_group/vote.cpp

```diff
@@ -82,8 +79,8 @@ int main() {
     std::cout << "Skipping test\n";
     return 0;
   }
-  check(Queue, 240, 80, 9, 8);
-  check(Queue, 24, 12, 9, 10);
-  check(Queue, 1024, 256, 9, 8);
+  check(Queue, 240, 80, 3, 1);
+  check(Queue, 24, 12, 3, 4);
+  check(Queue, 1024, 256, 3, 1);
```

Comment:
Why were these values changed?

Reply:
The initial design relies on the fact that sg_size is greater than R, otherwise, SG.get_local_id().get(0) % D == R is always false. But the minimal subgroup size for native subgroup is 4.

Reply:
Ok, thanks.  I'm not going to block this PR, but we should think about a way to rewrite some of these tests to reduce the number of magic constants.  Even just giving these better names than `D` and `R` could help readability.